assets/javascripts/openqa.js

```diff
@@ -577,19 +577,43 @@ function renderComments(row) {
 }
 
 function renderHttpUrlAsLink(value) {
-  const span = document.createElement('span');
-  for (let match; (match = value.match(/https?:\/\/[^\s,]*/)); ) {
-    const url = match[0];
-    const link = document.createElement('a');
-    link.href = url;
-    link.target = 'blank';
-    link.appendChild(document.createTextNode(url));
-    span.appendChild(document.createTextNode(value.substr(0, match.index)));
-    span.appendChild(link);
-    value = value.substr(match.index + url.length);
+  if (!value) {
+    return document.createTextNode('');
   }
-  span.appendChild(document.createTextNode(value));
-  return span;
+  if (Array.isArray(value)) {
+    const fragment = document.createDocumentFragment();
+    value.forEach((item, index) => {
+      fragment.appendChild(renderHttpUrlAsLink(item));
+      if (index < value.length - 1) {
+        fragment.appendChild(document.createTextNode(', ')); // rator between items
```

Comment:
what's a "rator"?

Reply:
Added `sepe` :sweat_smile: :+1: 
And yes makes sense that the git message has higher prio > PR message, i've improved the commit message and will keep that in mind. Also i will have a look at hub, thanks :slightly_smiling_face: